Android.common.mk

```diff
@@ -115,6 +115,9 @@ LOCAL_CPPFLAGS += \
 	-DENABLE_RBC
 endif
 
+LOCAL_CPPFLAGS += \
```

Comment:
This should have a board config, perhaps you can use ifeq ($(strip $(BOARD_CURSOR_WA)), true) for now.